nova/virt/vmwareapi/vm_util.py

```diff
@@ -1733,6 +1733,8 @@ def _process_host_stats(obj, host_reservations_map):
     # Total vcpus is the sum of all pCPUs of individual hosts
     # The overcommitment ratio is factored in by the scheduler
     threads = getattr(hardware_summary, "numCpuThreads", 0)
+    pcpus = getattr(hardware_summary, "numCpuCores", 0)
```

Comment:
Should we update the comment? It doesn't seem right to say `vcpus`, which is set to the value of `threads` is the sum of all `pCPUs` when we then get the `pcpus`.

Reply:
the comment was broken before this, for the same reason. but no reason not to fix it. done.